### ISSUE TITLE
Question about training Configuration in paper

### ISSUE THREAD

**Issue by ywh187** [ictnlp/LLaVA-Mini#15]
Hello, thank you for your great work!
The paper mentions that "LLaVA - Mini employs the same configurations as LLaVA v1.5 (Liu et al., 2023b)". I assumed that parameters like the number of epochs and the learning rate would be identical. However, upon checking Table 9, I found that during the finetuning phase, LLaVA - Mini was trained for 2 epochs, which differs from LLaVA 1.5 (it was trained for 1 epoch during the finetuning phase). Moreover, the learning rates also seem to be different. I'm curious to know how the authors define "the same configurations" in this context. 
Thanks!
